doc/source/data-ingest.rst

```diff
@@ -0,0 +1,140 @@
+.. _data-ingest:
+
+.. toctree::
+    :hidden:
+
+    conduce-entities
+
+====================
+Intro to Data Ingest
+====================
+
+Visualizing your data in Conduce begins with ingest.  Ingest is the process translating your data into a format that Conduce understands.  Conduce uses entities to define unique objects.  Entities are ingested into datasets.  Datasets hold collections of related entities and provide the interface a user needs to connect their data to a visualization renderer.
```

Comment:
"Ingest is the process translating your data" isn't quite right.
"process of", or "process that translates" would be better

Reply:
If you're going to use the term "visualization renderer" here, you need to explain the rendering concept somewhere.  Probably in a yet-to-be-written section defining terms and giving an overview of the conduce system

Reply:
Giving this a 2nd pass - is this too static dataset focused?  If I were to explain this to DHL talking about CMX data, I don't think this would make sense.  Can we remove the reference to entities.

Also, we reference samples below, but at this point in the docs, the user probably has not seen the entities page - we should give a quick definition for sample and entity

Reply:
Not sure about the overview comment.  I'm not going for completeness.  I do think it is important to hint at what comes next where appropriate.  However this is a data ingest intro, not an everything conduce can do manual.  I think we'll have to let "visualization renderers" be a mystery a little longer.

Reply:
I agree that this is choppy claw.  I'll see about rewording.  This document focuses on static entities because they are the simplest to ingest.  The conduce entities goes through an example of ingesting dynamic entities.